src/main/java/edu/umich/its/cpm/MigrationTaskService.java

```diff
@@ -305,6 +305,7 @@ private String getSiteName(String siteId, String sessionId) {
 					String.class);
 			JSONObject siteJSONObject = new JSONObject(siteJson);
 			siteName = (String) siteJSONObject.get("title");
+			siteName=Utils.sanitizeName(Utils.COLLECTION_TYPE,siteName);
```

Comment:
Yes Dave, other part of the code use this function call. It won't have negative effect on then it will make it better. Other parts of the code that is calling w.r.t to this function have similar bug exist. The fix is easy but i had to test whole bunch of other stuff to make sure everything is usually working